react-16.8-test-site/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "react16",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@yext/answers-headless-react": "file:..",
+    "react": "16.8.0",
+    "react-dom": "16.8.0",
```

Comment:
I think you have to do:
```
"react": "file:../node_modules/react",
"react-dom": "file:../node_modules/react-dom",
```
if you are linking answers-headless-react with "file:..", otherwise you get two copies of React and it may get an error like `Uncaught Invariant Violation: Invalid hook call.  blah blah. You might have more than one copy of React in the same app`

Reply:
I was relying on `--install-links` which avoids that issue, but I can update

Reply:
gotcha, yea this might just be easier if we have the package linking in package.json. Some people might forget to do --install-links and that would cause confusion